IBERTpy/latex/eyescan_summary.tex

```diff
@@ -0,0 +1,359 @@
+\documentclass{report}
+\usepackage{graphicx}
+\usepackage[a4paper, margin=0.5in]{geometry}
+\usepackage{subcaption}
+%\usepackage{printlen}
+\usepackage{pdfpages}
+%\uselengthunit{cm}
+%\usepackage{import}
+%\usepackage{xifthen}
+\usepackage{ifthen}
+\usepackage{pgf}
+\usepackage{tikz}
+%\usepackage{comment}
+\usepackage{intcalc}
+\usepackage{xcolor} % Required for specifying custom colours
+\definecolor{grey}{rgb}{0.9,0.9,0.9} % Colour of the box surrounding the title
+
+\newlength\imageheight
+\newlength\imagewidth
+
+\newcommand{\includegraphicsmaybe}[1]{\IfFileExists{#1}{\includegraphics[width=\linewidth]{#1}}{\includegraphics[width=\linewidth]{not_found.pdf}}}
+\newcommand\dateofscans{06-08-22} %Enter the date on which the scans were made (e.g. 06-13-22), assuming eyescans were properly filed
```

Comment:
consider replacing the date with `\today`

Reply:
While I could do that, the eyescans typically take all night to run so generating the summary pdf will typically be done on the next day.  As a result, if I use `\today` the script will not work if someone attempts to generate the summary pdf on a day different from the one on which the scans were run.

Reply:
suggest doing something like what is proposed here: https://stackoverflow.com/questions/2033844/passing-command-line-arguments-to-latex-document

`latex "\def\myvar{info-to-pass} \input{<filename>}"`

Reply:
Ok, your suggestion worked.  Now we do not need to modify the latex script every time we run it.  I also updated the README for this.